.stylelintrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     rules: {
         "block-no-empty": true,
         "color-hex-case": "lower",
-        "max-nesting-depth": 3,
+        "max-nesting-depth": 4,
```

Comment:
On avait dit à Antoine qu'on forçait `3` et qu'on ajoutait une exception où c'est nécessaire `/* stylelint-disable max-nesting-depth */`. Il faudra peut-être envisager à changer la règle si c'est trop fréquent et qu'on ne trouve pas d'alternative.